### PR TITLE
amd64: move the right parenthesis to the right place

### PR DESCRIPTION
**Pull request by wy-chung**
Like GPROC0_SEL, GUSERLDT_SEL also occupies two consecutive entries in segment descriptor table. So logically to specify the second descriptor of GUSERLDT_SEL, it shuld be (GUSERLDT_SEL  + 1).

### CODE REVIEW

**Comment by ehem** [comment]
The commit message needs to say more.  I'm wondering why the parentheses were used in the first place.  Older style?  Otherwise seems reasonable.

**Comment by wy-chung** [comment]
I have modified the commit message.

**Review by bsdimp — APPROVED** [review]

**Comment by bsdimp** [comment]
This looks good. Sorry it took me so long to review such a trivial and obviously right change

File: sys/amd64/amd64/machdep.c [hunk 1]
```diff
@@ -1382,7 +1382,7 @@ hammer_time(u_int64_t modulep, u_int64_t physfree)
 	 */
 	for (x = 0; x < NGDT; x++) {
 		if (x != GPROC0_SEL && x != (GPROC0_SEL + 1) &&
-		    x != GUSERLDT_SEL && x != (GUSERLDT_SEL) + 1)
+		    x != GUSERLDT_SEL && x != (GUSERLDT_SEL + 1))
```
Comment (gmshake) [inline sys/amd64/amd64/machdep.c:1385]: According to the precedence, the **un-fixed** one behaves the same with the **fixed** one. But I like the later form, which is more clear.

**Review by gmshake — APPROVED** [review]
Looks good to me.

**Comment by gmshake** [comment]
> Like GPROC0_SEL, GUSERLDT_SEL also occupies two consecutive entries in segment descriptor table. So logically to specify the second descriptor of GUSERLDT_SEL, it shuld be (GUSERLDT_SEL + 1).

There should be nothing bad happens. See my inline comment.

**Pull request merged**